testkit/nodekit/node.go

```diff
@@ -35,14 +41,30 @@ func NewNode(path string, tp node.Type, network string, cfg *nodebuilder.Config,
 	if err != nil {
 		return nil, err
 	}
-	store, err := nodebuilder.OpenStore(path)
+
+	keysPath := filepath.Join(path, "keys")
+	encConf := encoding.MakeConfig(app.ModuleEncodingRegisters...)
+	ring, err := keyring.New(app.Name, cfg.State.KeyringBackend, keysPath, os.Stdin, encConf.Codec)
+	if err != nil {
+		return nil, err
+	}
+
+	store, err := nodebuilder.OpenStore(path, ring)
 	if err != nil {
 		return nil, err
 	}
 	return nodebuilder.NewWithConfig(tp, p2p.Network(network), store, cfg, options...)
 
```

Comment:
could you please remove this blank line? 🙏 
(sorry for my OCD haha 🥲)

Reply:
Fixed 👍 
Thanks for the catch